src/savegame_json.cpp

```diff
@@ -1324,6 +1335,11 @@ void avatar::load( const JsonObject &data )
 {
     Character::load( data );
 
+    // Remove after 0.G
```

Comment:
```suggestion
    // TEMPORARY until 0.G
```
All-caps `TEMPORARY` is what we agreed at some point as the thing to be able to grep for on new releases; I realise that other comments here don't match that style.

Reply:
done